config/labels/analyzers/clangsa.json

```diff
@@ -1225,6 +1272,14 @@
       "profile:extreme",
       "profile:sensitive"
     ],
+    "unix.cstring.NotNullTerminated": [
```

Comment:
you can add it to the default profile as it only finds very basic cases. no false positives expected